### PR TITLE
Added ROUND function

### PR DESCRIPTION
**Pull request by abame**

### CODE REVIEW

File: tests/Oro/Tests/ORM/AST/Query/Functions/fixtures/mysql/round.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- functions:
+    - { name: "round", className: "Oro\\ORM\\Query\\AST\\Functions\\Numeric\\Round", type: "string" }
```
Comment (x86demon) [inline tests/Oro/Tests/ORM/AST/Query/Functions/fixtures/mysql/round.yml]: Please, fix type to numeric

File: tests/Oro/Tests/ORM/AST/Query/Functions/fixtures/postgresql/round.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+- functions:
+    - { name: "round", className: "Oro\\ORM\\Query\\AST\\Functions\\Numeric\\Round", type: "string" }
```
Comment (x86demon) [inline tests/Oro/Tests/ORM/AST/Query/Functions/fixtures/postgresql/round.yml]: Please, fix type to numeric

**Comment by x86demon** [comment]
Hi, @abame, great work!
Thank you for contribution.

**Comment by x86demon** [comment]
@abame , please, accept CLA, and I'll be able to merge this PR

**Comment by abame** [comment]
@x86demon where can i do that?

**Comment by x86demon** [comment]
https://www.orocrm.com/contributor-license-agreement

**Comment by abame** [comment]
@x86demon done

**Pull request merged**